src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -823,8 +825,8 @@ internal void TestFormulasParseRoundtrip(string script)
             {
                 AllowsSideEffects = false
             };
-            var userDefinitions = UserDefinitions.ProcessUserDefinitions(script, parserOptions, out var userDefinitionResult);
-            Assert.False(userDefinitionResult.HasErrors);
+            var parseResult = UserDefinitions.Parse(script, parserOptions);
+            Assert.False(parseResult.HasErrors);
         }
 
         internal void TestFormulasParseError(string script)
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

Not part of your changes, but this is only called inside this class, consider changing to private